src/ImageSegmentation.jl

```diff
@@ -3,9 +3,14 @@ module ImageSegmentation
 import Base: show
 
 using LinearAlgebra, Statistics
-using Images, DataStructures, StaticArrays, ImageFiltering, LightGraphs, SimpleWeightedGraphs, RegionTrees, Distances, StaticArrays, Clustering, MetaGraphs
+using DataStructures, StaticArrays, ImageCore, ImageFiltering, ImageMorphology, LightGraphs, SimpleWeightedGraphs, RegionTrees, Distances, StaticArrays, Clustering, MetaGraphs
 import Clustering: kmeans, fuzzy_cmeans
 
+accum(::Type{T}) where {T<:Integer} = Int
```

Comment:
This seems like a good chance to rename it to `accum_type`.

Reply:
I should keep it i core.jl right?

Reply:
I believe It's okay to just keep it here.